### PR TITLE
Added model card for neuraly/bert-base-italian-cased-sentiment

### PR DESCRIPTION
**Pull request by gianpy15**
Hello, we are making this pull request to add our Italian sentiment model to your repository of transformers.  
Thank you again for hosting the models :)

### CODE REVIEW

File: model_cards/neuraly/bert-base-italian-cased-sentiment/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---  
+language: it
+thumbnail: "https://neuraly.ai/static/assets/images/huggingface/thumbnail.png"
```
Comment (julien-c) [inline model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:3]: i think for social media the aspect ratio should be wider/more rectangle.

Maybe we should add guidelines somewhere (I know GitHub does it in repo settings)

File: model_cards/neuraly/bert-base-italian-cased-sentiment/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---  
+language: it
+thumbnail: "https://neuraly.ai/static/assets/images/huggingface/thumbnail.png"  
+tags:  
+  
+- pytorch  
+- tf
+- bert
+- text-classification
```
Comment (julien-c) [inline model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:9]:
```suggestion
```

You don't need those, they'll be added automatically

File: model_cards/neuraly/bert-base-italian-cased-sentiment/README.md (hunk 1 above)
Comment (ab3llini) [reply model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:3]: Oh ok cool! What about a 16:9 ratio? The URL remains the same though, its hosted on our website. Do you have any "template" reference?

Comment (ab3llini) [reply model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:3]: I changed the thumbnail with the attached image.
P.s.: The current version of the inference API is serving an older version of the model without labels. 
We fixed that, how long will it take before the API reloads the model config? Thank you 😊 

thumbnail

Comment (julien-c) [reply model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:3]: Looks awesome!

For the template, in any GitHub repo, you can look at the Settings > Social preview > Download template

Comment (julien-c) [reply model_cards/neuraly/bert-base-italian-cased-sentiment/README.md:3]: For the Inference API, should be updated now

**Pull request merged**